src/libraries/Common/src/Interop/Unix/System.Native/Interop.Threading.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal unsafe partial class Sys
+    {
+        [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_RuntimeThread_CreateThread")]
```

Comment:
This name looks odd. It should be just `SystemNative_CreateThread`.

Reply:
That one is in flight in https://github.com/dotnet/runtime/pull/62571. I would rather not do renaming before we have things building and did at least a single integration back to runtimelab (just in case we need the NuGet packages from there in the next couple weeks).